pkg/bpf/bpf.go

```diff
@@ -50,6 +50,7 @@ type BpfInfo struct {
 
 	Type       ebpf.ProgramType
 	AttachType ebpf.AttachType
+	VerLT513   bool
```

Comment:
seems we donot need to make this flag as a inner field

Reply:
yes, you are right